state/executor.go

```diff
@@ -218,31 +218,37 @@ func (e *Executor) BeginTxn(
 		PostHook:    e.PostHook,
 	}
 
-	// enable contract deployment allow list (if any)
-	if e.config.ContractDeployerAllowList != nil {
-		txn.deploymentAllowList = addresslist.NewAddressList(txn, contracts.AllowListContractsAddr)
+	// allow/block list should be possible if
```

Comment:
Finish comment.

Check if this is good enough:
```suggestion
	// contract deployment access list should are enabled either if access lists super admin is defined or access lists are defined in the genesis
```

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1839/commits/0387a8b41875f716ef490c2f24f2b2e2ad086266